Assets/Scripts/PlayerMovement.cs

```diff
@@ -98,7 +109,7 @@ private IEnumerator MoveBox(Vector3Int dir)
     /******* Rotate *******/
     public void ClockwiseRotate()
```

Comment:
Are the rotations not counted as a move?

Reply:
I have a seperate variable `NumberOfRotations` tracking this, in case we wanted to drill down. Number of moves + number of rotations gives the total moves made

Reply:
You are capturing the time between moves upon every call to MoveBox, but not for RotateBox. This is what I am confused about.

Reply:
Ah okay. Yeah I missed that, will make the change.

Reply:
@adamschachne fixed this